src/compiler/types.ts

```diff
@@ -1007,6 +1013,10 @@ namespace ts {
         expression: LeftHandSideExpression;
         typeArguments?: NodeArray<TypeNode>;
     }
+    export interface SupportedExpressionWithTypeArguments extends ExpressionWithTypeArguments {

```

Comment:
After read `isSupported...`, is `EntityNameExpressionWithTypeArguments` a better name? That looks like the property that is checked.

---
In reply to: [73189548](https://github.com/Microsoft/TypeScript/pull/10013#discussion_r73189548) [](ancestors = 73189548,72693405)